### PR TITLE
add file stratis-intro.md

### LINKED ISSUE #257: Intro Request on landing page
Hi, could you add an intro to the top of your website that is like the executive summary in the latex document?

copy&paste:

```
Stratis is a new tool that meets the needs of Red Hat Enterprise Linux (RHEL) users (among
others) calling for an easily configured, tightly integrated solution for storage that works within
the existing Linux storage management stack. To achieve this, Stratis prioritizes a straightforward
command-line experience, a rich API, and a fully automated, externally-opaque approach to
storage management. It builds upon elements of the existing storage stack as much as possible,
to enable delivery within 1-2 years. Specifically, Stratis initially plans to use device-mapper and
the XFS filesystem, but may incorporate other technology in the future
```

And below maybe 2-5 lines of shell code that show how it adds value?

Reason for asking: When I google for stratis storage, that is the first page that comes up. And if they are not as excited as me about RHEL tooling they might simply leave again as the page doesn't explain what the tool actually is.

Happy to help implement this, if desired.

### PR DESCRIPTION
**Pull request by drckeefe**
Closes #257

### CODE REVIEW

File: content/stratis-intro.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Learning and gaining experience in a number of different storage technologies
+can take many years.  Each of those technologies may have their own unique
+command-line syntax, APIs, options, and logging.  Stratis simplifies this by
+providing a single CLI and API for users to set up complex storage stacks
+without having to spend time learning each independent storage technology.
+
+### CLI Example ###
+
+The simplicity that Stratis provides can be seen when comparing the CLI
+commands used for create a filesystem that encrypts data-at-rest, to the
```
Comment (mulkieran) [inline content/stratis-intro.md:46]: "for create" needs to be restated.

File: content/stratis-intro.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+*Dennis Keefe, Stratis Team*
+
+### Stratis Description ###
+
+Stratis is a tool to easily configure pools and filesystems with enhanced
+storage functionality that works within the existing Linux storage
+management stack.  To achieve this, Stratis prioritizes a straightforward
+command-line experience, a rich API, and a fully automated approach to storage
+management. It builds upon elements of the existing storage stack as much as
+possible.  Specifically, Stratis uses device-mapper, LUKS, XFS, Clevis, and
```
Comment (mulkieran) [inline content/stratis-intro.md:18]: "XFS, Clevis" needs an "and"

File: content/stratis-intro.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+* device-mapper - A framework for logical to physical mapping of data blocks
+* LUKS          - An on disk format for encryption that can securely manages
+                multiple passwords
+* XFS           - A scalable, journaling, and performant filesystem
+* Clevis        - A framework for automated decryption
+
+Learning and gaining experience in a number of different storage technologies
+can take many years.  Each of those technologies may have their own unique
+command-line syntax, APIs, options, and logging.  Stratis simplifies this by
```
Comment (mulkieran) [inline content/stratis-intro.md:39]: It might not be clear what "this" refers to on this line.

File: content/stratis-intro.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+`pvcreate /dev/mapper/luks-device`
+
+`vgcreate -L100 -n lv1 vg1`
+
+`mkfs.xfs /dev/mapper/lv1-vg1`
+
+### API Example ###
+
+Stratis has also built an API for developers to implement the storage
```
Comment (mulkieran) [inline content/stratis-intro.md:79]: This is the _D-Bus_ API, better to be specific here.

File: content/stratis-intro.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Print version:
+
+`busctl get-property org.storage.stratis3 /org/storage/stratis3 org.storage.stratis3.Manager.r0 Version`
+
+Print engine state:
+
+`busctl call org.storage.stratis3 /org/storage/stratis3 org.storage.stratis3.Manager.r0 EngineStateReport”`
+
+Get ManageObjects:
```
Comment (mulkieran) [inline content/stratis-intro.md:103]: ^Manage^Managed

File: content/stratis-intro.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+`busctl call org.storage.stratis3 /org/storage/stratis3 org.freedesktop.DBus.ObjectManager GetManagedObjects`
+
+
+<!-- more -->
```
Comment (mulkieran) [inline content/stratis-intro.md:108]: Move this up to the location where you have "<Read more...>", @line 21. This annotation down here is interpreted by zola. "<Read more...>" is just ignored.

**Review by mulkieran — CHANGES_REQUESTED** [review]
This displays just fine and ends up where desired. The commands look a little odd. It would probably sort them out better by using a "> " at the start of the commands and a "# " at the start of the descriptions of commands.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Please see requested formatting changes here: https://github.com/stratis-storage/stratis-docs/pull/259. Other than that, I think it is ready. You will have to rebase to bring in recently added changes.

**Comment by drckeefe** [comment]
Closing. I will post new PR

**Pull request closed without merging**